addon/styles/addon.scss

```diff
@@ -40,47 +44,79 @@
 }
 
 .frost-list {
-	display: flex;
-	flex-direction: column;
-}
-
-.frost-list .container::before,
-.frost-list .container::after {
-	content: '';
-	position: absolute;
-	top: 0;
-	width: 1px;
-	height: 100%;
-	background-color: $frost-color-lgrey-1;
-}
-
-.frost-list .container::before {
-	left: 0;
-}
-
-.frost-list .container::after {
-  right: 0;
-}
-
-.frost-list .border-top {
-	height: 1px;
-  min-height: 1px;
-	margin-right: 15px;
-	background-color: $frost-color-lgrey-1;
-}
+  display: flex;
+  flex-direction: column;
+
+  .content {
```

Comment:
I've been pushing a new convention for this 

```
  .frost-list {
    &-content {
```

which results in `frost-list-content`, easier to keep it unique